Dockerfile

```diff
@@ -12,7 +12,7 @@ COPY bashrc /root/.bashrc
 WORKDIR /usr/src/app
 
 COPY package.json package-lock.json* ./
-RUN npm install
+RUN npm install --legacy-peer-deps
```

Comment:
the svelte ecosystem doesnt seem very caught up, so this was the only solution i could find to make the preview work